index.html

```diff
@@ -32,7 +32,7 @@
       </div>
       <nav class="desktop" id="nav">
         <div id="logo">
-          <a href="./home.html"><img src="./assets/dc-header-logo.png" alt="dc-logo"></a>
+          <a href="./index.html"><img src="./assets/dc-header-logo.png" alt="dc-logo"></a>
         </div>
         <div id="nav-links">
           <ul>
```

Comment:
- [ ] Nice work implementing the Navbar, it looks really good :+1: :sunglasses: However It would be nice to stick to the provided design as much as possible. In the provided design the two bars content are align on the right site, kindly make sure you follow that.
### Your design :point_down: 
![Screenshot from 2023-05-25 10-49-13](https://github.com/badger-99/Capstone-Project-1/assets/91142432/a418df4a-dc0b-4042-970e-e37c933e5de2)
### What it is supposed to look like :point_down: 
![Screenshot from 2023-05-25 10-56-35](https://github.com/badger-99/Capstone-Project-1/assets/91142432/aed1172b-3188-4e8d-ab46-49451dcb9877)

_**HIT:** Updating the padding-right of `#nav` to use the same value of `#top-bar` padding-right (`padding: 0.25rem 6rem;`) will solve our problem_ :smiley_cat: